prow/cluster/plank_deployment.yaml

```diff
@@ -29,13 +29,22 @@ spec:
         role: prow
       containers:
       - name: plank
-        image: gcr.io/k8s-prow/plank:0.17
+        image: gcr.io/k8s-prow/plank:0.18
+        args:
+        - "--dry-run=false"
+        - "--github-bot-name=k8s-ci-robot"
```

Comment:
@kargakis please go for it, it's a lot easier here than over in mungegithub, I've been stuck behind all the globals over there